nav2_smac_planner/include/nav2_smac_planner/a_star.hpp

```diff
@@ -22,6 +22,7 @@
 #include <memory>
 #include <queue>
 #include <utility>
+#include <tuple>
```

Comment:
You can remove the last paragraph and the code snippet from the readme if this is added.

Usual stuff on docs updates + the smac planner readme has a param guide inline, so updating that too. 

I imagine this makes things much slower, it would be good to have that boldly spelled out that this should never be enabled outside of debugging

Reply:
Still this stuff